systems/billing/collector/pkg/config.go

```diff
@@ -49,6 +50,9 @@ func NewConfig(name string) *Config {
 				"event.cloud.local.{{ .Org}}.subscriber.simmanager.package.activate",
 				"event.cloud.local.{{ .Org}}.subscriber.simmanager.sim.allocate",
 				"event.cloud.local.{{ .Org}}.dataplan.package.package.create",
+
+				// "event.cloud.global.{{ .Org}}.orchestrator.package.package.create",
```

Comment:
I guess this comment should be removed.

Reply:
The context for this comment is updated.